contracts/GameJam.sol

```diff
@@ -1,11 +1,42 @@
 pragma solidity 0.5.11;
 
 contract GameJam {
+  enum Stages {
+        JamInProgress,
+        WinnerDeclaration,
+        Finished
```

Comment:
Is there to be more than 3 stages, maybe a registration phase for adding Competitors ? 
Any stages need renaming?

Reply:
Three is perfect for the moment, we can add more stages later on.

Maybe:

1. `Registration`
1. `InProgress`
1. `Finished`

wdyt?

Reply:
These three make sense. Under what condition should we move from Registration to InProgress? Should I implement a `start()` function ?

Reply:
Please <3

Reply:
Added a start function and restricted it to Registration stage.

Reply:
super ⭐️